### ISSUE TITLE
[BUG] - Slider won't move, wireguard tunnel won't connect

### ISSUE THREAD

**Issue by wave864** [zaneschepke/wgtunnel#249]
**Describe the bug**
The slider to the right of each tunnel stays locked in the off position in WG Tunnel and will not move.

**Smartphone (please complete the following information):**

- Device: Samsung Galaxy S23 (SM-S911B/DS)
- Android Version: 14 (One UI 6.1)
- App Version: 3.4.6

**To Reproduce**

1. Go to 'Tunnel'
2. Click on any slider to activate a tunnel.
3. Slider stays in the off position.

https://github.com/zaneschepke/wgtunnel/assets/53859987/c1a837aa-5d44-459b-9ac9-c1ac1062b592

**Expected behavior**
Tapping the slider should connect the tunnel.

**Additional context**
I still have the official WireGuard app installed and it continues to work without problems. I tried both importing tunnels that function without issues in the official WireGuard app and creating a completely new tunnel, the issue occured in both cases. Installed the app from the Play Store. Unlike the official app, WG Tunnel also does not show up in the VPN section of the Android settings.

**Comment by zaneschepke** [comment]
Hello! By chance, was your VPN running when you tried this? Please turn off your VPN in the official app first and then try again.

**Comment by wave864** [comment]
Hi! So this is the weirdest thing.

I did try it with both VPN on and off in the official app, reinstalled WG Tunnel several times, reimported the tunnels, nothing changed. This went on for two days, the only thing I did not do was delete the official wg app because I need a working tunnel. 

Now, just after reading your message, I tried one more time, literally changing nothing, and all of a sudden I get the system popup warning me of an attempt to create a vpn connection and now it just works. WG Tunnel also shows up as a VPN in the system settings now. I wish I could be more helpful here but I am stumped. I guess I can consider my issue solved, but there was _something_ going on.
